src/ray/gcs/gcs_server/gcs_job_manager.cc

```diff
@@ -224,8 +224,8 @@ void GcsJobManager::HandleGetAllJobInfo(rpc::GetAllJobInfoRequest request,
                 const Status &status,
                 const rpc::NumPendingTasksReply &num_pending_tasks_reply) {
               if (!status.ok()) {
-                RAY_LOG(ERROR) << "Failed to get is_running_tasks from core worker: "
-                               << status.ToString();
+                RAY_LOG(WARNING) << "Failed to get is_running_tasks from core worker: "
+                                 << status.ToString();
```

Comment:
hmm, if the status is not ok, should we early return, seems the current code keeps executing.

Reply:
Let's not handle it in this PR and make a follow-up? This error message actually impacts the user experience, so let's make sure we merge this PR first. cc @architkulkarni to follow up with this comment (we probably want to fix that by 2.6 as well).

Reply:
@jjyao @rkooo567 In the case of returning early from status not ok for one of the jobs: 
- should we set is_running_tasks to True or False?
- should we set the status in the overall reply (which contains info for all the jobs) to OK, or something else?

Also, why should it be a 2.6 release blocker? Will it crash if it doesn't return early?

Reply:
I think we should reply OK but have a special flag to indicate the result is not known.

Reply:
the ERROR log is a release blocker. This can easily happen under common cases,  and ERROR logs will make  drivers spammy. I already saw this message frequently from CI tests.